parser/sql/dialect/mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -872,7 +872,7 @@ predicate
     | bitExpr NOT? BETWEEN bitExpr AND predicate
     | bitExpr SOUNDS LIKE bitExpr
     | bitExpr NOT? LIKE simpleExpr (ESCAPE simpleExpr)?
-    | bitExpr NOT? REGEXP bitExpr
+    | bitExpr NOT? (REGEXP | RLIKE) bitExpr
```

Comment:
Since RLIKE and REGEXP are synonyms, this change is ok.

https://github.com/mysql/mysql-server/blob/ea7087d885006918ad54458e7aad215b1650312c/sql/lex.h#L610